src/api-reference/budget/budget-adjustments.md

```diff
@@ -18,34 +34,34 @@ This resource is used to add budget adjustments. Each budget item detail may hav
 
 4.0
 
-## <a name=“post”></a> Create a list of Budget Adjustments
+## <a name="post"></a> Create a list of Budget Adjustments
 
     POST /budget/v4/adjustments
 
 #### Parameters
 
-Name | Type | Format | Description
----|---|---|---
-`useMonthlyRollingUpdate`|`boolean`|`query`|If true, all adjustments for a given month, adjustment type, amount type & description will be rolled up to one adjustment.
+| Name                      | Type      | Format  | Description                                                                                                                 |
```

Comment:
This table formatting contains too much whitespace. Please fix and use the markup formatting found in https://github.concur.com/developer/api/blob/master/documentation-template.md